Dockerfile.prod

```diff
@@ -47,14 +47,17 @@ RUN pip3 install setuptools==36.0.1
 # Node
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
     apt-get install -y nodejs && rm -rf /var/lib/apt/lists/*
-COPY ./package.json /code/listenbrainz/
-COPY ./package-lock.json /code/listenbrainz/
+RUN mkdir /static
+WORKDIR /static
+COPY ./package.json /static
+COPY ./package-lock.json /static
 RUN npm install
+COPY ./listenbrainz/webserver/static /static
 
 COPY . /code/listenbrainz
+RUN npm run build:prod
```

Comment:
I tried in the container directly, and the web pages are working just fine.

So I think all the JS material can be deleted (node_modules folder, jsx source files, package-* files, webpack files)

Reply:
note that you'd have to do this in a single RUN command to save layers - `RUN npm install && npm run build... && rm -r node_modules`, otherwise it won't save space. Probably better to push for a multi-stage build